cirq/ops/raw_types.py

```diff
@@ -362,10 +355,10 @@ def transform_qubits(self, func: Callable[[Qid], Qid]) -> 'Operation':
         """
         return self.with_qubits(*(func(q) for q in self.qubits))
 
-    def controlled_by(self,
-                      *control_qubits: Qid,
-                      control_values: Optional[Sequence[
-                          Union[int, Collection[int]]]] = None) -> 'Operation':
+    def control(self,
```

Comment:
I don't know if it makes sense to rename Operation.controlled_by.  This will break code that doesn't use partially specified ControlledGates.